src/SFA.DAS.EAS.Employer_Financial.Database/Scripts/PreDeployment/AML-2873_TransferPaymentDate.sql

```diff
@@ -0,0 +1,61 @@
+﻿IF COL_LENGTH('employer_financial.TransactionLine', 'Id') IS NULL
+BEGIN
+BEGIN TRY
+BEGIN TRANSACTION
+    CREATE TABLE [employer_financial].[TransactionLine2]
+	(
+		Id BIGINT PRIMARY KEY IDENTITY(1,1) NOT NULL,
+		AccountId BIGINT NOT NULL,
+		DateCreated DATETIME NOT NULL,
+		SubmissionId BIGINT NULL,
+		TransactionDate DATETIME NOT NULL,
+		TransactionType TINYINT NOT NULL DEFAULT 0, 
+		LevyDeclared DECIMAL(18,4) NULL, 
+		Amount DECIMAL(18,4) NOT NULL DEFAULT 0, 
+		EmpRef nVarchar(50) null,
+		PeriodEnd nVarchar(50) null,
+		Ukprn BIGINT null, 
+		SfaCoInvestmentAmount DECIMAL(18, 4) NOT NULL DEFAULT 0, 
+		EmployerCoInvestmentAmount DECIMAL(18, 4) NOT NULL DEFAULT 0,
+		[EnglishFraction] DECIMAL(18, 5) NULL, 
+		[TransferSenderAccountId] BIGINT NULL, 
+		[TransferSenderAccountName] NVARCHAR(100) NULL,
+		[TransferReceiverAccountId] BIGINT NULL, 
+		[TransferReceiverAccountName] NVARCHAR(100) NULL
+	)
+	
+
+INSERT INTO [employer_financial].[TransactionLine2] SELECT * FROM [employer_financial].[TransactionLine]
+
+
+EXEC sp_rename 'employer_financial.TransactionLine', 'TransactionLine3'
+EXEC sp_rename 'employer_financial.TransactionLine2', 'TransactionLine'
+
+
+CREATE INDEX [IX_TransactionLine_SubmissionId] ON [employer_financial].[TransactionLine] (SubmissionId)
+CREATE INDEX [IX_TransactionLine_AccountId] ON [employer_financial].[TransactionLine] (AccountId) INCLUDE (Ukprn,PeriodEnd,TransactionType)
+CREATE INDEX [IX_TransactionLine_Payment] on [employer_financial].[TransactionLine] (PeriodEnd,AccountId,Ukprn,TransactionDate, DateCreated)
+CREATE UNIQUE INDEX [IX_TransactionLine_TransactionType_SubmissionId] ON [employer_financial].[TransactionLine] (SubmissionId) WHERE (TransactionType = 1);
+CREATE INDEX [IX_TransactionLine_AccountId_DateCreated] ON [employer_financial].[TransactionLine] (AccountId, DateCreated) WITH (ONLINE = ON)
```

Comment:
Is there a reason for creating these indexes as online? This script will be running during an upgrade, it's a new table (so won't be accessed by anything yet) so creating as an offline index be faster and use less disk space?